docs/root/_static/searchtools.js

```diff
@@ -0,0 +1,584 @@
+/*
+ * searchtools.js
+ * ~~~~~~~~~~~~~~~~
+ *
+ * Sphinx JavaScript utilities for the full-text search.
+ *
+ * :copyright: Copyright 2007-2020 by the Sphinx team, see AUTHORS.
+ * :license: BSD, see LICENSE for details.
+ *
+ */
+
+// Modified from https://raw.githubusercontent.com/sphinx-doc/sphinx/3.x/sphinx/themes/basic/static/searchtools.js
+// to have renderApiVersionLabel to render the API version for each search result item.
```

Comment:
@dio could you do a baseline PR to check in the original searchtools.js, that way we can review just the delta here?

Reply:
Got it. https://github.com/envoyproxy/envoy/pull/13008.